src/psyclone/tests/test_files/dynamo0p3/infrastructure/operator_mod.f90

```diff
@@ -62,14 +62,33 @@ module operator_mod
 
   type, extends(base_operator_type) ::  columnwise_operator_type
      integer :: c
+  contains
+     procedure, public :: get_proxy => get_proxy_columnwise
   end type columnwise_operator_type
 
   type, extends(base_operator_type) ::  columnwise_operator_proxy_type
-     integer :: d
+     integer(i_def) :: alpha
```

Comment:
I hadn't realised we had no compilation tests for CMA! Thanks for adding this stub infrastructure.